lib/openstack/openstack_handle/handle.rb

```diff
@@ -60,9 +63,15 @@ def self.raw_connect(username, password, auth_url, service = "Compute", extra_op
       raise
     end
 
-    def self.auth_url(address, port = 5000, ssl = false)
-      scheme = ssl ? "https" : "http"
-      "#{scheme}://#{address}:#{port}/v2.0/tokens"
+    def self.auth_url(address, port = 5000, scheme = "http")
+      url(address, port, scheme)
+    end
+
+    def self.url(address, port = 5000, scheme = "http", path = "/v2.0/tokens")
+      port = port.to_i
```

Comment:
There are `auth_url` and `raw_connect` methods in EmsOpenstackMixin.  I'd like to remove those altogether, since I can't find any callers for those methods.  But, I'd rather not tie that to this PR.  

In any case, EmsOpenstackMixin.auth_url calls OpenstackHandle::Handle.auth_url.  And, until we delete that, I'm not sure what can actually be passed in here.  Plus, this is a public class method.  Even if I pretend that all the callers are always passing in ints, I can't be sure that's what's always going to happen ... e.g., Automate?